R/read_AHCCD_daily_precip.R

```diff
@@ -0,0 +1,158 @@
+#' Reads AHCCD daily precipitation file
+#'
+#' @description This program reads Adjusted and Homogenized Canadian Climate Data (AHCCD) 
+#' file of rainfall, snowfall or total precipitation. The values are arranged as
+#' month x day, which makes them difficult to read using standard \R functions.
+#' @param precip_file Required. Name of the file to be read.
+#' @return If successful, returns the values in a data frame, consisting of the \code{date}, 
+#' the precipitation variable and the data \code{code}. The name of the precipitation variable, \code{rain}, 
+#' \code{snow} or \code{total} is inferred from the file name.
+#' If unsuccessful, returns the value \code{FALSE}.
+#' @author Kevin Shook
+#' @seealso  \code{\link{read_AHCCD_daily_temp}} \code{\link{read_AHCCD_monthly_precip}} 
+#' @references
+#'Monthly AHCCD data are available from \url{http://www.ec.gc.ca/dccha-ahccd}. 
+#'Daily values must be requested. Any use of the precipitation data must cite 
+#'\cite{Mekis, E and L.A. Vincent, 2011: An overview of the second generation 
+#'adjusted daily precipitation dataset for trend analysis in Canada. 
+#'Atmosphere-Ocean, 49 (2), 163-177.}
+#' @examples
+#' \dontrun{
+#'stoon_rainfall <- read_AHCCD_daily_precip("dr_4057120.txt")}
+#' @export
+
+read_AHCCD_daily_precip <- function(precip_file){
+  # check parameter
+  if (precip_file == "" | is.null(precip_file)){
+    cat('Error: precip_file missing\n')
```

Comment:
I think you want `stop` here. That's what I would use.

Reply:
Good idea. Will do.